static/js/components/ChannelSidebar_test.js

```diff
@@ -0,0 +1,19 @@
+import React from "react"
+import ReactMarkdown from "react-markdown"
+import { assert } from "chai"
+import { shallow } from "enzyme"
+
+import ChannelSidebar from "./ChannelSidebar"
+
+import { makeChannel } from "../factories/channels"
+
+describe("ChannelSidebar", () => {
+  const renderSidebar = channel => shallow(<ChannelSidebar channel={channel} />)
+
+  it("should render sidebar", () => {
+    const channel = makeChannel()
+    const wrapper = renderSidebar(channel)
+    const [description] = wrapper.find(ReactMarkdown)
+    assert.equal(description.props.source, channel.description)
+  })
```

Comment:
a basic test that "There is no description..." shows up when `channel.description` is empty might be worthwhile here